cdap-ui/app/cdap/components/EntityCard/index.js

```diff
@@ -49,6 +50,21 @@ export default class EntityCard extends Component {
     }
   }
 
+  renderJumpButton() {
+    const entity = this.props.entity;
+    if (!['datasetinstance', 'stream'].includes(entity.type) && !entity.isHydrator) {
```

Comment:
nit: `Array.includes` browser support starts from chrome 47. We had an issue with chromium in Ubuntu VMs. We need to make sure we have appropriate polyfills